README.md

```diff
@@ -182,6 +182,7 @@ App|Description
 [dev_hid_composite](usb/device/dev_hid_composite) | Composite HID (mouse + keyboard) example from TinyUSB
 [dev_hid_generic_inout](usb/device/dev_hid_generic_inout) | Generic HID device example from TinyUSB
 [dev_lowlevel](usb/device/dev_lowlevel) | A USB Bulk loopback implemented with direct access to the USB hardware (no TinyUSB)
+[dev_midi](usb/device/dev_midi) | USB midi device example from TinyUSB
```

Comment:
I wonder if it's worth mentioning here that this just sends dummy note data to the host PC? (i.e. it isn't (yet) a fully-fledged USB MIDI hardware interface) :man_shrugging:

Reply:
Good idea, I've updated the readme.